components/form.js

```diff
@@ -1,125 +1,122 @@
-import React from 'react';
-import Link from 'next/link';
-import { fetchBearerToken, fetchFormFields, postFormData } from './fetchCalls';
+import React from "react";
+import Link from "next/link";
+import { fetchBearerToken, fetchFormFields, postFormData } from "./fetchCalls";
 
 const baseUrl = process.env.baseUrl;
 
 export default class Form extends React.Component {
-    constructor() {
-        super();
-        this.state = {
-            token: {
-                accessToken: '',
-                expiresIn: 0,
-                refreshToken: '',
-                tokenType: ''
-            },
-            formFields: [],
-            formData: {}
-        };
-    }
-
-    // Make state targets equal to the value of input fields
-    change = e => {
-        this.setState({
-            formData: {
-                ...this.state.formData,
-                [e.target.name]: e.target.value
-            }
-        });
+  constructor() {
+    super();
+    this.state = {
+      token: {
+        accessToken: "",
+        expiresIn: 0,
+        refreshToken: "",
+        tokenType: ""
+      },
+      formFields: [],
+      formData: {}
     };
+  }
 
-    isTokenValid() {
-        let isValid = false;
+  // Make state targets equal to the value of input fields
+  change = e => {
+    this.setState({
+      formData: {
+        ...this.state.formData,
+        [e.target.name]: e.target.value
+      }
+    });
+  };
 
-        const currentTime = new Date().getTime();
-        const tokenFetchTime = this.state.token.fetching_time;
-        const expirationTime = this.state.token.expires_in;
+  isTokenValid() {
+    let isValid = false;
 
-        if (currentTime < tokenFetchTime + expirationTime) {
-            isValid = true;
-        }
+    const currentTime = new Date().getTime();
```

Comment:
this PR is to update the readme, why is this file changing?

Reply:
I was working on this branch when I made changes in this file. Sorry!